lang/en.yml

```diff
@@ -379,7 +379,7 @@ en:
     PASSWORD: Password
     PASSWORDEXPIRED: 'Your password has expired. Please choose a new one.'
     PLURALNAME: Members
-    REMEMBERME: 'Remember me next time?'
+    REMEMBERME: 'Remember me next time? (for %d days on this device)'
```

Comment:
I would prefer if this just said *"Keep me signed in"*. I don't think we need to reveal how long a token will last for and a specific number is misleading because if a user regularly visits the site, they might not have to log in again for much longer than we indicate here. Conversely if they log out on a different device tomorrow, they'll be logged out on this device as well (by default) much sooner than `%d` days. You could argue that this is exposing configuration too, particularly if `%d` represents a long time it could highlight the site as being easier to attack.

If we do want to give users more detailed information about how the login system works maybe we can use a tooltip?